src/components/validations/validation-icon.style.js

```diff
@@ -4,25 +4,37 @@ import StyledIcon from '../icon/icon.style';
 import { isClassic } from '../../utils/helpers/style-helper';
 
 const ValidationIconStyle = styled.div`
-  background: none;
-  cursor: pointer;
-  display: flex;
-  align-items: center;
-  padding: 1px;
-  margin-top: 0;
-  margin-bottom: 0;
-  margin-left: 8px;
-  border: 0;
-  outline: none;
+  ${({ theme }) => css`
+    background: none;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    padding: 1px;
+    margin-top: 0;
+    margin-bottom: 0;
+    margin-left: 8px;
+    border: 0;
+    outline: none;
 
-  ${StyledIcon}:before {
-    color: ${({ validationType, theme }) => theme.colors[validationType]};
-  }
+    ${!isClassic(theme) && css`
```

Comment:
I thinking about moving it to the bottom to keep everything in one `isClassic` condition and just remove the outline if inside the classic theme. If you will do that you can also extract whole condition to an external 'classic' file